src/Wordpress/CustomPostType/Timeframe.php

```diff
@@ -792,7 +792,7 @@ protected function getCustomFields() {
 			),
 			array(
 				'name'    => esc_html__( "Grid", 'commonsbooking' ),
-				'desc'    => esc_html__( 'Choose whether users can only select the entire from/to time period when booking (full slot) or book within the time period in an hourly grid. See the documentation: <a target="_blank" href="https://commonsbooking.org/?p=437">Manage Booking Timeframes</a>', 'commonsbooking' ),
+				'desc'    => commonsbooking_sanitizeHTML( 'Choose whether users can only select the entire from/to time period when booking (full slot) or book within the time period in an hourly grid. See the documentation: <a target="_blank" href="https://commonsbooking.org/?p=437">Manage Booking Timeframes</a>', 'commonsbooking' ),
```

Comment:
@hansmorb Wenn ich nichts übersehen habe, können wir das so nicht lassen, weil auf diesem Weg der Text nicht übersetzt wird.

Reply:
@hansmorb Das müsste geändert werden in:
`commonsbooking_sanitizeHTML( __( 'Choose whether users can only select the entire from/to time period when booking (full slot) or book within the time period in an hourly grid. See the documentation: <a target="_blank" href="https://commonsbooking.org/?p=437">Manage Booking Timeframes</a>', 'commonsbooking' )),`